frontend/src/app/components/comparison-chart/comparison-chart.component.ts

```diff
@@ -0,0 +1,99 @@
+import {Component, OnInit} from '@angular/core';
+import {CovidService} from '../../services/covid.service';
+import {SocketService} from '../../services/socket/socket.service';
+import {SelectItem, TreeNode} from 'primeng/api';
+import {ChartModelBuilder} from '../../model/chart-model-builder';
+import {TreeTable} from 'primeng/treetable';
+
+@Component({
+  selector: 'app-comparison-chart',
+  templateUrl: './comparison-chart.component.html',
+  styleUrls: ['./comparison-chart.component.scss'],
+})
+export class ComparisonChartComponent implements OnInit {
```

Comment:
I think the name comparison chart is a little bit too unspecific. We will have multiple charts which will compare something. Maybe call it CompareRegionsChart or something like that